src/syncEnrichers.ts

```diff
@@ -0,0 +1,75 @@
+import { EnricherInput } from './codecs';
+import { GraphQLClient } from 'graphql-request';
+import { ENRICHERS, CREATE_ENRICHER, UPDATE_ENRICHER } from './gqls';
+import { RequestAction } from '@transcend-io/privacy-types';
+import { Identifier } from './fetchIdentifiers';
+
+/**
+ * Sync an enricher configuration
+ *
+ * @param enricher - The enricher input
+ * @param client - GraphQL client
+ * @param identifiersByName - Index of identifiers in the organization
+ */
+export async function syncEnricher(
+  enricher: EnricherInput,
+  client: GraphQLClient,
+  identifiersByName: { [name in string]: Identifier },
+): Promise<void> {
+  // Try to fetch an enricher with the same title
+  const {
+    enrichers: { nodes: matches },
+  } = await client.request<{
```

Comment:
Does this need to paginate over all of the possible matching enrichers in case there are multiple pages of matches?

Reply:
it could, here its looking for a direct match on title and pretty unlikely there would be more than 10 entries. think its fine to bake this assumption in for now, and potentially adjust the GraphQL API in the future to be more performant with this use case